README.md

```diff
@@ -25,53 +25,157 @@
 
 ---
 
-REAFLOW is a modular diagram engine for build static or interactive editors.
-The library is feature rich and modular allowing for displaying complex visualizations
-with total customizability.
+REAFLOW is a modular diagram engine for building static or interactive editors. The library is feature-rich and modular allowing for displaying complex
+visualizations with total customizability.
 
 ## 🚀 Quick Links
-- Checkout the [docs and demos](https://reaflow.dev)
-- Learn about updates from the [Changelog](CHANGELOG.md)
+
+- :point_right: Checkout the [**docs and demos**](https://reaflow.dev)
+- :newspaper: Learn about updates from the [Changelog](CHANGELOG.md)
 - Explore the library on [Chroma](https://www.chromatic.com/library?appId=5f99ba42fe88ac0022fd1147)
 
 ## 📦 Usage
-Install the package via NPM:
+
+Install the package via **NPM**:
 
 ```
 npm i reaflow --save
 ```
 
-then add the component with some nodes and edges:
+Install the package via **Yarn**:
+
+```
+yarn add reaflow
+```
+
+## Usage with Create React App (no SSR support, client-side support only)
+
+Add the component with some nodes and edges:
 
 ```jsx
 import React from 'react';
 import { Canvas } from 'reaflow';
 
 export default () => (
-  <Canvas
-    nodes={[
-      {
-        id: '1',
-        text: '1'
-      },
-      {
-        id: '2',
-        text: '2'
-      }
-    ]}
-    edges={[
-      {
-        id: '1-2',
-        from: '1',
-        to: '2'
-      }
-    ]}
-  />
+  <div style={{ position: 'relative', width: '100vw', height: '100vh' }}>
```

Comment:
Let's keep this example just as simple as possible. Let's remove these wrapper divs

Reply:
I don't think it's a wise choice. The default example is too simple and if you just take it as it is you won't see the nodes because the canvas width/height won't be set. I had to figure this out myself when beginning with reaflow.

I think those default provide a better developer experience.

Reply:
For the readme, i strongly believe it needs to be bare bones. I understand your concern but i'd rather that be in the real docs.

Reply:
I still disagree. At the very least, it should be documented about being in a component with specified width/height. I think that's the least that should be done, because people have the tendency to copy/paste the code in the readme and expecting it to show something. And it won't, leading to a bad first experience with the lib.

We can either have it as-it, or specify in a comment that it should have a width/height, or we can do nothing.

I spent 15mn on my first try figuring this out, things should be dead-simple.

Reply:
I get your point but the readme is really just the 'sales' pitch and general info. Users should use the docs to get beyond that. Let's remove this and LGTM.